### PR TITLE
added language codes and fix small errors

### PR DESCRIPTION
**Pull request by stefangrotz**

### CODE REVIEW

**Comment by stefangrotz** [comment]
Small fix for #3155

File: docs/COMMUNITIES.md [hunk 1]
```diff
@@ -15,8 +15,15 @@ Many language communities are self organising and have their contact channels on
    * [Web](https://dhivehi.ai/)
    * [Discord](https://discord.com/invite/5ZdTHsN8Mn)
    * [Matrix](https://app.element.io/#/room/!WNiJFJTBpvYqMrDWir:matrix.org)
+* Esperanto (`eo`):
+   * [Matrix](https://chat.mozilla.org/#/room/#Esperanto:mozilla.org)
+   * [GitHub](https://github.com/parolrekonado)
+   * [Website](https://parolrekonado.github.io/)
 * French (`fr`):
```
Comment (MichaelKohler) [inline docs/COMMUNITIES.md:22]: Is this the correct link?

File: docs/COMMUNITIES.md [hunk 2]
```diff
@@ -15,8 +15,15 @@ Many language communities are self organising and have their contact channels on
    * [Web](https://dhivehi.ai/)
    * [Discord](https://discord.com/invite/5ZdTHsN8Mn)
    * [Matrix](https://app.element.io/#/room/!WNiJFJTBpvYqMrDWir:matrix.org)
+* Esperanto (`eo`):
+   * [Matrix](https://chat.mozilla.org/#/room/#Esperanto:mozilla.org)
+   * [GitHub](https://github.com/parolrekonado)
+   * [Website](https://parolrekonado.github.io/)
```
Comment (MichaelKohler) [inline docs/COMMUNITIES.md:21]: Is this the correct link?

Comment (phirework) [reply docs/COMMUNITIES.md:21]: Looks correct to me, what are you concerned about?

Comment (MichaelKohler) [reply docs/COMMUNITIES.md:21]: When I opened it 2 days ago it showed the theme documentation, but there indeed was work done recently. All good now!

**Pull request merged**